fix typo in LCD mode timing description (#464)
gbdev/pandocs: src/STAT.md

Mode | Action                                                      | Duration                                                           | Accessible video memory
-----|------------------------------------------------------------------|--------------------------------------------------------------------|-------------------------

>   2  | Searching OAM for OBJs whose Y coordinate overlap this line | 80 dots                                                    | VRAM, CGB palettes
>   3  | Reading OAM and VRAM to generate the picture                | 168 to 291 dots, depending on sprite count            | None
>   0  | Nothing (HBlank)                               | 85 to 208 dots, depending on previous mode 3 duration | VRAM, OAM, CGB palettes
>   1  | Nothing (VBlank)                                 | 4560 dots (10 scanlines)                                  | VRAM, OAM, CGB palettes

Reviewer: ```suggestion
  2  | Searching OAM for OBJs whose Y coordinate overlap this line | 80 dots                                               | VRAM, CGB palettes
  3  | Reading OAM and VRAM to generate the picture                | 168 to 291 dots, depending on sprite count            | None
  0  | Nothing (HBlank)                                            | 85 to 208 dots, depending on previous mode 3 duration | VRAM, OAM, CGB palettes
  1  | Nothing (VBlank)                                            | 4560 dots (10 scanlines)                              | VRAM, OAM, CGB palettes
```

Mode | Action                                                      | Duration                                                           | Accessible video memory
-----|------------------------------------------------------------------|--------------------------------------------------------------------|-------------------------

>   2  | Searching OAM for OBJs whose Y coordinate overlap this line | 80 dots (19 µs)                                                    | VRAM, CGB palettes

Reviewer: ```suggestion
  2  | Searching OAM for OBJs whose Y coordinate overlap this line | 80 dots (20 µs)                                                    | VRAM, CGB palettes
```

>   3  | Reading OAM and VRAM to generate the picture                | 168 to 291 dots (40 to 69 µs) depending on sprite count            | None
>   0  | Nothing (HBlank)                               | 85 to 208 dots (20 to 49 µs) depending on previous mode 3 duration | VRAM, OAM, CGB palettes

Reviewer: ```suggestion
  3  | Reading OAM and VRAM to generate the picture                | 168 to 291 dots (42 to 73 µs) depending on sprite count            | None
  0  | Nothing (HBlank)                               | 85 to 208 dots (21 to 52 µs) depending on previous mode 3 duration | VRAM, OAM, CGB palettes
```
